README.md

```diff
@@ -727,6 +729,39 @@ hi AerialGuide2 guifg=Blue
 
 <!-- /API -->
 
+## TreeSitter queries
+
+When writing queries, the following captures and metadata are used by Aerial:
+
+- `@type` - **required** capture for the logical region being captured
+- `kind` - **required** metadata, a string value matching one of `vim.lsp.protocol.SymbolKind`
+- `@name` - capture to extract a name from its text
+- `@start` - a start of the match, influences matching of cursor position to aerial tree, defaults to `@type`
+- `@end` - an end of the match, influences matching of cursor position to aerial tree, defaults to `@start`
+- `@selection` - position to jump to when using Aerial for navigation, only used if `treesitter.experimental_selection_range` is enabled, falls back to `@name` and `@type`
```

Comment:
I've also just shipped the selection range option, so there is no flag and this is always enabled